api/GlobalEventHandlers.json

```diff
@@ -947,10 +947,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.5"
+              "version_added": "9"
```

Comment:
There's a lot of changes from 3.5. Can you check where that data came from? Maybe it was supported on one of the other interfaces?

Reply:
This data came from the initial wiki migration.  Doing some additional testing around, I couldn't find any interface that supported these event handlers, so it's probably that the event is supported in Firefox 3.5, but not the event handlers.

Reply:
Yes, that's a common situation.